### PR TITLE
[NP-7784] Load funds flow cleanup

### PR DESCRIPTION
**Pull request by thepinkcacao**
nano: https://github.com/nanoPayinc/NANOPAY/pull/15361
Ticket: https://nanopay.atlassian.net/browse/NP-7784
Jenkins CI: https://ci-jenkins.nanopay.net/view/CI/job/ci-NP-7784-Load_Funds_Flow_Cleanup_/1/console (SUCCESS)
Description:

- Load funds flow cleanup including  UI and bug fix
- Add Javascript implementation for next scheduled dates
- Add notes based on the  Schedule options(One time deposit or Schedule deposit)
- Add a static note for the review wizard

Images: 
1. Fund my Balance(One time Deposit and Schedule deposit)
scheduleOptions

2. Next scheduled dates(Year)
nextScheduledDate_year

3. Next scheduled dates(Month: ON_THE or EACH)
nextScheduledDate_month2
nextScheduledDate_month

4. Next scheduled dates(Week)
nextScheduledDate_week

5. Next scheduled dates(Day)
nextScheduledDate_day

6. review( Schedule Deposit and One time deposit  )
Screen Shot 2022-10-04 at 11 29 42 AM

### CODE REVIEW

File: src/foam/u2/wizard/wizardlet/SuccessWizardlet.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ foam.CLASS({
           })
         ];
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'isLastWizardlet',
+      value: false
```
Comment (kgrgreer) [inline src/foam/u2/wizard/wizardlet/SuccessWizardlet.js]: not needed, false is the default value:

File: src/foam/u2/wizard/wizardlet/ReviewWizardletView.js [hunk 2]
```diff
@@ -84,6 +87,9 @@ foam.CLASS({
                   })
                   .start(item.headingBorder)
                     .tag(item.view, {data: self.data.value[item.name]})
+                    .startContext({ data: self.data?.value?.[item?.name] })
+                      .tag(item.view)
+                    .endContext()
```
Comment (sarthak-marwaha) [inline src/foam/u2/wizard/wizardlet/ReviewWizardletView.js]: This seems to be breaking a few wizardlets, below suggestion should fix that
```suggestion
                    .tag(item.view, { ...(
                      self.data.value[item.name] ?
                      {data: self.data.value[item.name]} :
                      {}
                    )})
```

File: src/foam/u2/wizard/controllers/IncrementalWizardController.js [hunk 3]
```diff
@@ -54,6 +57,11 @@ foam.CLASS({
       class: 'Boolean',
       value: false
     },
+    {
+      class: 'Boolean',
+      name: 'showTitle',
+      value: true
```
Comment (sarthak-marwaha) [inline src/foam/u2/wizard/controllers/IncrementalWizardController.js]: This should be false and set on the wizardlet instead

File: src/foam/u2/detail/RowPropertyView.js [hunk 4]
```diff
@@ -31,7 +38,7 @@ foam.CLASS({
       this
         .addClass()
         .start()
-          .add(this.prop.label)
+          .add(this.prop.columnLabel).show(this.prop.columnLabel != '' ? true : false)
```
Comment (sarthak-marwaha) [inline src/foam/u2/detail/RowPropertyView.js]:
```suggestion
          .add(this.prop.columnLabel).show(this.prop.columnLabel)
```

File: src/foam/u2/PropertyBorder.js [hunk 5]
```diff
@@ -184,9 +184,9 @@
            */
           addClass(this.myClass('errorText')).
           enableClass(this.myClass('colorText'), colorSlot).
-          // show(errorSlot.and(modeSlot.map(m => m == foam.u2.DisplayMode.RW))).
+           show(errorSlot.and(modeSlot.map(m => m == foam.u2.DisplayMode.RW))).
```
Comment (sarthak-marwaha) [inline src/foam/u2/PropertyBorder.js:193]: Why this change?

File: src/foam/nanos/cron/SimpleIntervalScheduleView.js [hunk 6]
```diff
@@ -27,6 +27,9 @@ foam.CLASS({
     ^halfWidth {
       grid-column: span 6;
     }
+    ^halfWidth.foam-u2-PropertyBorder {
+      justify-content: flex-start;
```
Comment (sarthak-marwaha) [inline src/foam/nanos/cron/SimpleIntervalScheduleView.js:31]: Why is this required?

File: src/foam/u2/PropertyBorder.js (hunk 5 above)
Comment (thepinkcacao) [reply src/foam/u2/PropertyBorder.js:193]: because there is too much space on fund your balance view (below the choices and above from) because of the error text space.

File: src/foam/nanos/cron/SimpleIntervalScheduleView.js (hunk 6 above)
Comment (thepinkcacao) [reply src/foam/nanos/cron/SimpleIntervalScheduleView.js:31]: @sarthak-marwaha because I tried to align these two fields.
Screen Shot 2022-10-05 at 12 17 34 PM
